eventcatalog/tsconfig.json

```diff
@@ -19,6 +19,7 @@
     },
     "jsx": "react-jsx",
     "jsxImportSource": "react",
-    "types": ["vitest/globals"]
+    "types": ["vitest/globals"],
+    "lib": [ "dom", "dom.iterable", "es2016", "esnext" ]
```

Comment:
Hey @simonwfarrow 

Thanks for the PR.

Any reason we had to change the tsconfig?